node/sync/src/controllers/peers.rs

```diff
@@ -64,6 +67,17 @@ impl SyncPeers {
         }
     }
 
+    pub fn states(&self) -> HashMap<PeerState, u64> {
+        let mut states: HashMap<PeerState, u64> = HashMap::new();
+
+        for info in self.peers.values() {
```

Comment:
can this be written as self.peers.values().iter().map()?

Reply:
Either way is ok.